packages/replayio/src/utils/graphql/fetchUserIdFromGraphQLOrThrow.ts

```diff
@@ -25,9 +25,7 @@ export async function fetchUserIdFromGraphQLOrThrow(accessToken: string) {
           }
         }
           `,
-    {
-      key: accessToken,
-    },
```

Comment:
`accessToken` is not a variable but it's part of the header, we were passing it in both places here